lambdas/service/app.py

```diff
@@ -735,3 +740,66 @@ def _file_url(uuid, **kwargs):
     return (config.service_endpoint() + '/dss/files' +
             '/' + urllib.parse.quote(uuid, safe='') +
             '?' + urllib.parse.urlencode(kwargs))
+
+
+@app.route('/url', methods=['POST'], cors=True)
+def shorten_query_url():
+    """
+    Take a URL as input and return a (potentially) shortened URL that will redirect to the given URL
+
+    parameters:
+        - name: url
+          in: body
+          type: string
+          description: URL to shorten
+
+    :return: A 200 response with JSON body containing the shortened URL:
+
+    ```
+    {
+        "url": "http://shortened.url/ABC"
```

Comment:
We can probably have a more realistic example now that we have a custom domain name.